numba/targets/arraymath.py

```diff
@@ -1415,6 +1414,21 @@ def check_dimensions(array_like, name):
             if isinstance(array_like.key[0].key[0], types.Sequence):
                 raise TypeError("{0} has more than 2 dimensions".format(name))
 
+@register_jitable
+def _handle_ddof(ddof):
+    if not np.isfinite(ddof):
+        raise TypeError('Cannot convert non-finite ddof to integer')
+    if ddof - int(ddof) != 0:
+        raise TypeError('ddof must be integer')
```

Comment:
Same here, `ValueError`? Also, perhaps `ddof must be an integral value` as floating point is being accepted?

Reply:
Done